### PR TITLE
fix(card_Css): #2732 fix ant card css

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

**Review by evans-g-crsj — APPROVED** [review]

File: src/components/UserDashboard/UserDashboard.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  .ant-card-head-title {
+    padding: 0;
+  }
+
+  .ant-card-head {
+    padding: 0;
+  }
+
+  .ant-card-head-title > h3 {
+    font-family: 'Open Sans';
```
Comment (francisl) [inline src/components/UserDashboard/UserDashboard.scss:11]: theme <-

File: src/style/dist/themes/default/antd.css [hunk 2]
```diff
@@ -17329,12 +17329,12 @@ textarea.ant-pagination-options-quick-jumper input {
 }
 .ant-popover-inner-content {
   padding: 12px 16px;
-  color: rgba(0, 0, 0, 0.65);
+  color: #515885;
```
Comment (francisl) [inline src/style/dist/themes/default/antd.css:17332]: use color variable. from _colors.less

File: src/style/dist/themes/default/antd.css [hunk 3]
```diff
@@ -17329,12 +17329,12 @@ textarea.ant-pagination-options-quick-jumper input {
...
 }
 .ant-popover-message {
   position: relative;
   padding: 4px 0 12px;
-  color: rgba(0, 0, 0, 0.65);
+  color: #515885;
```
Comment (francisl) [inline src/style/dist/themes/default/antd.css:17337]: ☝️

File: src/theme/_vars.scss [hunk 4]
```diff
@@ -14,6 +14,8 @@ $lightBlue: #008199;
 $orange: #ffaa52;
 $purple: #a6278f;
 
+$greyScale11: #a9adc0;
```
Comment (francisl) [inline src/theme/_vars.scss:17]: ??? `/theme` is deprecated. Nothing should be added here

**Review by francisl — CHANGES_REQUESTED** [review]

**Review by francisl — APPROVED** [review]

**Pull request merged**